wgengine/filter/match.go

```diff
@@ -47,12 +47,18 @@ func (npr NetPortRange) String() string {
 	return fmt.Sprintf("%v:%v", npr.Net, npr.Ports)
 }
 
+type CapMatch struct {
```

Comment:
This name reuse really threw me off in the code review and I had to open it in VSCode to actually figure out what was going on.